tests/php/Storage/AssetStoreTest.php

```diff
@@ -278,55 +278,55 @@ public function dataProviderFileIDs()
                 'directory/2a17a9cb4b/file.jpg',
                 [
                     'Filename' => 'directory/file.jpg',
-                    'Hash' => sha1('puppies'),
+                    'Hash' => substr(sha1('puppies'), 0, 10),
```

Comment:
Why did this change from full sha to shortened?

Reply:
The original hash wasn't being use in the test, so it didn't matter what the hash was. My test needs a sensible hash, so I had to normalise it to what will actually be used in production

Reply:
FYI there's probably some super edge casy scenario where we end up with a collision when someone uploads a file whose sha1 hash has the same 10 character as some other file.